src/test/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/SimpleSvTypeInferenceAndConstructionUnitTest.java

```diff
@@ -0,0 +1,186 @@
+package org.broadinstitute.hellbender.tools.spark.sv.discovery;
+
+import com.google.common.collect.ImmutableSet;
+import htsjdk.variant.variantcontext.Allele;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+import static org.broadinstitute.hellbender.tools.spark.sv.discovery.SVDiscoveryTestDataProvider.*;
+import static org.broadinstitute.hellbender.tools.spark.sv.discovery.SimpleSVType.TYPES.*;
+import static org.broadinstitute.hellbender.tools.spark.sv.discovery.SimpleSVType.createBracketedSymbAlleleString;
+import static org.broadinstitute.hellbender.tools.spark.sv.discovery.SvTypeInference.inferFromNovelAdjacency;
+import static org.broadinstitute.hellbender.tools.spark.sv.utils.GATKSVVCFConstants.*;
+
+
+public class SimpleSvTypeInferenceAndConstructionUnitTest extends BaseTest {
```

Comment:
After https://github.com/broadinstitute/gatk/pull/3475, this should extend `GATKBaseTest`